documentation/IDTA-01002-3/modules/ROOT/pages/changelog.adoc

```diff
@@ -13,7 +13,7 @@ Note: Changes in Metamodel (IDTA-01001) will not be listed here, although they h
 
 Major Changes:
 
-* 
+* Extending the ServiceSpecificationProfileEnum to include all v3.1 profiles and correcting the example (https://github.com/admin-shell-io/aas-specs-api/issues/473[#473])
```

Comment:
What seems to be missing in changelog V3.0.4 vs. 3.1 is the profile https://industrialdigitaltwin.io/aas-specifications/IDTA-01002/v3.1/http-rest-api/service-specifications-and-profiles.html#concept-description-repository-service-specification-service-specification-ssp-002

Reply:
ok they are mentioned in a single line